easybuild/tools/github.py

```diff
@@ -324,6 +325,22 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    if not os.path.isdir(develop):
+        raise EasyBuildError("Downloading of develop branch failed: not found in %s" % develop)
+    try:
+        os.chdir(develop)
```

Comment:
@Caylo this chdir is the reason why the robot path has to be an absolute path rather than a relative path

This can be fixed by resolving the path specified to `--robot` to an absolute path before continuing, but you should `chdir` back to the location where you were before after applying the patch.

Or better yet, there's probably no reason the `chdir` yourself at all, since that's exactly what the 2nd argument you pass to `apply_patch` is for (if that's not working, you're hitting a bug)...